docs/ispc.rst

```diff
@@ -2958,6 +2964,17 @@ to the additional features they introduce.
     __attribute__((unmangled)) void foo(int a, int b);
 
 
+remove_ispc_copy
+----------------
+
+``__attribute__((remove_ispc_copy))`` can be applied to a function with
+``export`` qualifier. It informs the compiler that it should not generate a
```

Comment:
I tried to find a place in the docs where we explain that `export` function produces 2 variants and I can't find any mention of it. It would be helpful to include this information, and the 'Functions and Function Calls' section seems like a good place for it. It would also be useful to add a link to this section here

Reply:
I have added explanation about two function versions to the mentioned section.